src/PKSim.Core/Model/Simulation.cs

```diff
@@ -169,6 +169,12 @@ public virtual void AddUsedBuildingBlock(UsedBuildingBlock usedBuildingBlock)
 
       public virtual void AddReactions(ReactionBuildingBlock reactionBuildingBlock) => _allReactions.Add(reactionBuildingBlock);
 
+      public virtual void UpdateReactions(IEnumerable<ReactionBuildingBlock> reactionBuildingBlocks)
```

Comment:
We were having an interesting side effects where we kept adding to the list of reactions.

Reply:
interesting. never noticed that :)

Reply:
This is new. Until now, we only had one . In v12, it can be multiple (coming from different modules)